### ISSUE TITLE
Hold CTRL and drag to place a control point (CP) and continue a slider

### ISSUE THREAD

**Issue by RaiLTC** [ppy/osu#11873]
This is something frustrating at the moment. Where you have to place a CP, drag the tail CP to continue a slider and then reposition the placed CP. This basically eliminates that by placing a CP at the original position of the dragged CP. Saves time and easy to use!

**Comment by bdach** [comment]
I don't know if it's just me, but I have no idea what this is proposing.

**Comment by RaiLTC** [comment]
It's kind of hard to explain, so here is a [video](https://puu.sh/Hj3xh/7f04a06298.mp4) showing the frustrating part.

By holding CTRL and dragging it will basically place a control point at the original location and allow you to continue

**Comment by peppy** [comment]
I'll take a stab and say that you are wanting the ability to add a slider control point after the last control point, but can't right now in lazer (or stable)?

**Comment by RaiLTC** [comment]
Yes. Create the point on CTRL drag instead of manually doing it. It's more of a QoL thing
